### DISCUSSION TITLE
YouTube i reklamy propagandowe

### DISCUSSION THREAD

**Discussion by PolishRoboDog** [FiltersHeroes/KAD#1988] [Pokaż i opowiedz]
### Link bezpośredni 
`https://www.sb.by/en/`
### Dodatkowe informacje mogące mieć znaczenie
Wiem o tym że już nie blokujecie stron z fake newsami, ale może zrobicie wyjątek dla Łukaszenki który postanowił sobie wykupić reklamy na YouTube.

https://imgur.com/a/LakvFlR
https://pl.wikipedia.org/wiki/Sowietskaja_Bie%C5%82orussija

Zgłoszenie opublikowane anonimowo przez użytkownika.

**Comment by krystian3w** [comment]
Brak uruchomionej polonizacji wpisów blogowych z propagandą.

**Comment by blocker999** [comment]
@hawkeye116477 Nie wiem czy odrzucenie tego tematu to dobry ruch - bo ja szczerze mam wątpliwości tzn. portal nie ma polonizacji to fakt ale gościu prosi o blokade w ramch wyjątku a poza tym pasek informacyjny na filmiku jest po polsku i reklama 
"propagandowa" jest wyświetlana na terytorium Polski więc może to do @MajkiIT dać.

Nie chodzi o blokade białoruskiego serwisu a o same reklamy na YouTube: 

![BIA YT](https://user-images.githubusercontent.com/23176832/144590683-7c1a0f5e-4bd6-4b6d-beda-0b64bd270f37.png)

**Comment by krystian3w** [comment]
Reklamy YT są poza skillem MajkiIT, ktoś może celowo odblokował sobie je, `ryanbr` w liście Fanboy Annoyances regularnie testuje nowe pomysły z `json-prune` lub w przeglądarce Brave i ich blokerem reklam.

Przy czym filtr skryptowy oparty o json-prune co jakiś czas robi kuku ze stroną YT (pusty homepage i inne takie tam bugi).

> https://github.com/AdguardTeam/AdguardFilters/issues/101275#issuecomment-984642362 czy https://github.com/AdguardTeam/AdguardFilters/issues/101275#issuecomment-981107041

**Comment by hawkeye116477** [comment]
W zasadzie YT to serwis międzynarodowy, więc jeżeli rzeczywiście są tam takie reklamy i nie są blokowane, to raczej takie sprawy do EasyList albo gorhilla.

**Comment by krystian3w** [comment]
Opcjonalnie zgłoszenie spod 3 kropek i uzasadnienie że to fake news przepuszczony przez automatykę Google.


Ale to może wymagać dezaktywacji blokady reklam zamiast liczenia,  że mi wyciekają 2 na dzień - to któraś  będzie tą o Łukaszence (raczej widzę jakieś zalando, allegro, samochody, fast food).

**Comment by AroKol78** [comment]
> reklama "propagandowa" jest wyświetlana na terytorium Polski więc może to do

MSZ/ABW (niech reagują)

a tak poza tym myślałem, że reklamy na YT są blokowane przez filtry AG/uBO

**Comment by krystian3w** [comment]
Wyciekają codziennie, jedynie nie śledzę mega wątku AdGuard:

https://github.com/AdguardTeam/AdguardFilters/issues/101275#issuecomment-985449926

Na ile winne jest za dużo subskrybcji w AdGuard lub praca na 100 otwartych kart to nie wiem.
